src/main/java/net/mcreator/ui/blockly/BlocklyJavascriptBridge.java

```diff
@@ -213,6 +234,10 @@ private String[] openStringEntrySelector(Function<Workspace, String[]> entryProv
 			case "configuredfeature" -> openDataListEntrySelector(
 					w -> ElementUtil.loadAllConfiguredFeatures(w).stream().filter(e -> e.isSupportedInWorkspace(w))
 							.toList(), "configured_features");
+			case "global_triggers" -> openKeyAndValueEntrySelector(

```

Comment:
It may make more sense and require less code to collect ext_triggers into list of DataListEntry's and use openDataListEntrySelector I think.

Reply:
Oh, I didn't realize that that you can set a readable name for `DataListEntry`

Reply:
Yep, looks super clean now :)